ReactWindows/ReactNative.Tests/Modules/Network/NetworkingModuleTests.cs

```diff
@@ -119,23 +119,24 @@ public void NetworkingModule_Request_Content_String_NoContentType()
                 { "string", "Hello World" },
             };
 
-            var passed = true;
+            var passed = false;
             var waitHandle = new AutoResetEvent(false);
             var module = CreateNetworkingModule(new DefaultHttpClient(), new MockInvocationHandler((name, args) =>
             {
+                waitHandle.Set();
+
                 if (name != "emit" || args.Length != 2 || ((string)args[0]) != "didCompleteNetworkResponse")
                 {
                     return;
                 }
 
                 var array = args[1] as JArray;
-                if (array == null || array.Count != 2)
+                if (array == null || array.Count != 3)
```

Comment:
In that case, how about
```
private void OnRequestError(int requestId, string message)
{
    EventEmitter.emit("didCompleteNetworkResponse", new JArray
    {
        requestId,
        message
    });
}

private void OnRequestErrorDueToTimeout(int requestId, string message)
{
    EventEmitter.emit("didCompleteNetworkResponse", new JArray
    {
        requestId,
        message,
        true
    });
}
```

to save the bool check *and* the unnecessary bool arg? : ) Open to suggestions for naming `OnRequestErrorDueToTimeout`.